util/src/main/resources/org/killbill/billing/util/ddl.sql

```diff
@@ -234,14 +234,16 @@ CREATE TABLE bus_events_history (
 
 drop table if exists sessions;
 create table sessions (
-  record_id serial unique
-, start_timestamp datetime not null
-, last_access_time datetime default null
-, timeout int
-, host varchar(100) default null
-, session_data mediumblob default null
-, primary key(record_id)
+  record_id serial unique,
+  id varchar(36) NOT NULL,
```

Comment:
Nit: we prefer the style with a leading comma instead. It would also make the diff smaller ;-)